source/common/router/route_config_update_impl.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+#include "envoy/api/v2/route/route.pb.h"
+#include "envoy/router/rds.h"
+#include "envoy/router/route_config_update.h"
+
+#include "common/common/logger.h"
+#include "common/protobuf/utility.h"
+
+namespace Envoy {
+namespace Router {
+
+class RouteConfigUpdateImpl : public RouteConfigUpdate {
+public:
+  RouteConfigUpdateImpl(TimeSource& time_source) : time_source_(time_source) {}
+
+  bool onRdsUpdate(const envoy::api::v2::RouteConfiguration& rc,
```

Comment:
For the overrides, we typically group them separated by a blank line and then put `// Router::RouteConfigupdate` or something like that above. E.g. see https://github.com/envoyproxy/envoy/blob/3abb7d051c55f190d19b531a2fd1360842b334a5/source/common/http/conn_manager_impl.h#L66 for an example.